README.md

```diff
@@ -364,56 +408,63 @@ santa makedict --help
 santa makedict "./validation/participants.json"
 ```
 
-### Continuous Integration
+## Continuous Integration
+
 Continuous Integration (CI) aims to keep state updated to always match the code currently checked in a repository.
 This typically includes a build, automated test runs, and possibly making sure that the newly built artifacts are
 deployed to a target environment. This helps developers and users by providing timely feedback and showing what the
 results of certain checks were on a given version of the code.
 
-This repository uses [Travis CI](https://travis-ci.com) to run tests automatically when new commits are pushed. Results
```

Comment:
we should probably keep something here like `This repository uses GitHub Actions...` and point to the `python-package.yml` file

Reply:
I have added some links and descriptions, let me know what you think